Description: set the infobars type to "other" rather than "question"
 to avoid blue cluebars
Bug-Ubuntu: https://launchpad.net/bugs/1259059
Index: nautilus-3.18.1/src/nautilus-trash-bar.c
===================================================================
--- nautilus-3.18.1.orig/src/nautilus-trash-bar.c
+++ nautilus-3.18.1/src/nautilus-trash-bar.c
@@ -232,6 +232,6 @@ nautilus_trash_bar_new (NautilusFilesVie
 {
 	return g_object_new (NAUTILUS_TYPE_TRASH_BAR,
 			     "view", view,
-			     "message-type", GTK_MESSAGE_QUESTION,
+			     "message-type", GTK_MESSAGE_OTHER,
 			     NULL);
 }
Index: nautilus-3.18.1/src/nautilus-x-content-bar.c
===================================================================
--- nautilus-3.18.1.orig/src/nautilus-x-content-bar.c
+++ nautilus-3.18.1/src/nautilus-x-content-bar.c
@@ -388,7 +388,7 @@ nautilus_x_content_bar_new (GMount *moun
 			    const char * const *x_content_types)
 {
 	return g_object_new (NAUTILUS_TYPE_X_CONTENT_BAR,
-			     "message-type", GTK_MESSAGE_QUESTION,
+			     "message-type", GTK_MESSAGE_OTHER,
 			     "mount", mount,
 			     "x-content-types", x_content_types,
 			     NULL);
Index: nautilus-3.18.1/src/nautilus-special-location-bar.c
===================================================================
--- nautilus-3.18.1.orig/src/nautilus-special-location-bar.c
+++ nautilus-3.18.1/src/nautilus-special-location-bar.c
@@ -162,7 +162,7 @@ GtkWidget *
 nautilus_special_location_bar_new (NautilusSpecialLocation location)
 {
 	return g_object_new (NAUTILUS_TYPE_SPECIAL_LOCATION_BAR,
-			     "message-type", GTK_MESSAGE_QUESTION,
+			     "message-type", GTK_MESSAGE_OTHER,
 			     "special-location", location,
 			     NULL);
 }
